packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -203,6 +207,8 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
     releaseSchedule.releasePeriod = releasePeriod;
     releaseSchedule.releaseCliff = releaseStartTime.add(releaseCliffTime);
     releaseSchedule.releaseStartTime = releaseStartTime;
+    // Nonvalidators (employees) opt-out of Escheat, validators are opt-in.
```

Comment:
nit: "Expiry is opt-in for folks who can validate, opt-out for folks who cannot. This is because folks who are running Validators or Groups are likely to want to keep cGLD in the ReleaseGold contract even after it becomes withdrawable."